doc/reference/configuration/configuration_reference.rst

```diff
@@ -3,13 +3,280 @@
+
+-   :ref:`groups.\<group_name\> <configuration_reference_groups_name>`
+-   :ref:`groups.\<group_name\>.replicasets <configuration_reference_groups_name_replicasets>`
+-   :ref:`groups.\<group_name\>.\<config_parameter\> <configuration_reference_groups_name_config_parameter>`
+
+.. _configuration_reference_groups_name:
+
+.. confval:: groups.<group_name>
+
+    A group name.
+
+
+.. _configuration_reference_groups_name_replicasets:
+
+.. confval:: groups.<group_name>.replicasets
+
+    Replica sets that belong to this group. See :ref:`replicasets <configuration_reference_replicasets>`.
+
+
+.. _configuration_reference_groups_name_config_parameter:
+
+.. confval:: groups.<group_name>.<config_parameter>
+
+    Any configuration parameter that can be defined in the group :ref:`scope <configuration_scopes>`.
+    For example, :ref:`iproto <configuration_reference_iproto>` and :ref:`database <configuration_reference_database>` configuration parameters defined at the group level are applied to all instances in this group.
+
+
+
+..  _configuration_reference_replicasets:
+
+replicasets
+~~~~~~~~~~~
+
+.. NOTE::
+
+    ``replicasets`` can be defined in the group :ref:`scope <configuration_scopes>` only.
+
+-   :ref:`replicasets.\<replicaset_name\> <configuration_reference_replicasets_name>`
+-   :ref:`replicasets.\<replicaset_name\>.leader <configuration_reference_replicasets_name_leader>`
+-   :ref:`replicasets.\<replicaset_name\>.bootstrap_leader <configuration_reference_replicasets_name_bootstrap_leader>`
+-   :ref:`replicasets.\<replicaset_name\>.instances <configuration_reference_replicasets_name_instances>`
+-   :ref:`replicasets.\<replicaset_name\>.\<config_parameter\> <configuration_reference_replicasets_name_config_parameter>`
+
+.. _configuration_reference_replicasets_name:
+
+.. confval:: replicasets.<replicaset_name>
+
+    A replica set name.
+
+
+.. _configuration_reference_replicasets_name_leader:
+
+.. confval:: replicasets.<replicaset_name>.leader
+
+    A replica set leader.
+    This option is in effect when ``manual`` :ref:`replication.failover <configuration_reference_replication_failover>` is used.
+
+    **Example**
+
+    ..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/manual_leader/config.yaml
+        :language: yaml
+        :start-at: replicaset001:
+        :end-at: listen: 127.0.0.1:3303
+        :dedent:
+
+
+.. _configuration_reference_replicasets_name_bootstrap_leader:
+
+.. confval:: replicasets.<replicaset_name>.bootstrap_leader
+
+    A bootstrap leader for a replica set.
+    To specify a bootstrap leader manually, you need to set :ref:`replication.bootstrap_strategy <configuration_reference_replication_bootstrap_strategy>` to ``config``.
+
+    **Example**
+
+    ..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/bootstrap_strategy/config.yaml
+        :language: yaml
+        :start-at: replicaset001
+        :end-at: listen: 127.0.0.1:3303
+        :dedent:
```

Comment:
Can we highlight the lines with the `boostrap_strategy` and the `boostrap_leader` options?

BTW, why `failover: election` is here? On the website it looks like it is included intentionally, but I guess that it isn't.

Reply:
> BTW, why failover: election is here? On the website it looks like it is included intentionally, but I guess that it isn't.

Am I correct that it's is better to move `failover: election` at the global level to show only `replication. bootstrap_strategy` here?

Reply:
That's correct. Thanks!